test/erc-20-extensions/ERC20Extensions.js

```diff
@@ -0,0 +1,204 @@
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
Not the correct header.

Hmm seems the repo has fallen out of compliance. All js files should have license headers like this
https://github.com/hashgraph/hedera-smart-contracts/blob/main/test/multicall/Multicall.js